lib/util/serve-mvt.cjs

```diff
@@ -20,13 +20,38 @@ async function serveMvt(req, res) {
 
   // We use pickBy to remove empty layers
   const pbf = vtpbf.fromGeojsonVt(pickBy(layersTiles))
+  const pbfCompressed = await gzip(Buffer.from(pbf))
+  return pbfCompressed
+}
+
+const getPBFTileCompressed = async (layersFeatures, z, x, y) => {
+  // Generate cache key for redis
+  const cacheKey = `${z}:${x}:${y}`
+  // Check if tile buffer is already in cache
+  const cachedTile = await redis.getBuffer(cacheKey)
+  if (cachedTile) {
+    return cachedTile
+  }
+
+  // Generate tile buffer and store it in cache
+  const pbfTileCompressed = await generatePBFTileCompressed(layersFeatures, z, x, y)
+
+  // Store in cache for 1 hour
+  await redis.set(cacheKey, pbfTileCompressed, 'EX', 3600)
```

Comment:
Maybe prefer a env-var for the duration (something like MAPTILES_CACHE_DURATION=3600)